src/main/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestClientInternal.java

```diff
@@ -113,6 +120,9 @@
   Meter uploadThroughput; // Throughput for uploading blobs
   Meter inputThroughput; // Throughput for inserting into the Arrow buffer
 
+  // JVM and thread related metrics
+  MetricRegistry jvmAndThreadMetrics;
```

Comment:
can we change it to `jvmMetrics` or `jvmMemoryAndThreadMetrics`?

Reply:
oh yes, `jvmMemoryAndThreadMetrics` is better.